docs/spell_linked_spell.md

```diff
@@ -6,70 +6,17 @@
 
 This table provides data for spell linking system, telling it which spells trigger what, and under which conditions.
 
-<table>
-<colgroup>
-<col width="12%" />
-<col width="12%" />
-<col width="12%" />
-<col width="12%" />
-<col width="12%" />
-<col width="12%" />
-<col width="12%" />
-<col width="12%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td><p><strong>Field</strong></p></td>
-<td><p><strong>Type</strong></p></td>
-<td><p><strong>Attributes</strong></p></td>
-<td><p><strong>Key</strong></p></td>
-<td><p><strong>Null</strong></p></td>
-<td><p><strong>Default</strong></p></td>
-<td><p><strong>Extra</strong></p></td>
-<td><p><strong>Comment</strong></p></td>
-</tr>
-<tr class="even">
-<td><p><a href="#spell_trigger">spell_trigger</a></p></td>
-<td><p>mediumint(8)</p></td>
-<td><p>signed</p></td>
-<td><p> </p></td>
-<td><p>NO</p></td>
-<td><p>NULL</p></td>
-<td><p> </p></td>
-<td><p> </p></td>
-</tr>
-<tr class="odd">
-<td><p><a href="#spell_effect">spell_effect</a></p></td>
-<td><p>mediumint(8)</p></td>
-<td><p>signed</p></td>
-<td><p> </p></td>
-<td><p>NO</p></td>
-<td><p>0</p></td>
-<td><p> </p></td>
-<td><p> </p></td>
-</tr>
-<tr class="even">
-<td><p><a href="#type">type</a></p></td>
-<td><p>smallint(3)</p></td>
-<td><p>unsigned</p></td>
-<td><p> </p></td>
-<td><p>NO</p></td>
-<td><p>0</p></td>
-<td><p> </p></td>
-<td><p> </p></td>
-</tr>
-<tr class="odd">
-<td><p><a href="#comment">comment</a></p></td>
-<td><p>text</p></td>
-<td><p>signed</p></td>
-<td><p> </p></td>
-<td><p>NO</p></td>
-<td><p>NULL</p></td>
-<td><p> </p></td>
-<td><p> </p></td>
-</tr>
-</tbody>
-</table>
+| Field              | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|--------------------|--------------|------------|-----|------|---------|-------|---------|
+| [spell_trigger][1] | mediumint(8) | signed     |     | NO   | NULL    |       |         |
+| [spell_effect][2]  | mediumint(8) | signed     |     | NO   | 0       |       |         |
+| [type][3]          | smallint(3)  | unsigned   |     | NO   | 0       |       |         |
+| [comment][4]       | text         | signed     |     | NO   | NULL    |       |         |
```

Comment:
fix it for all text and varchar.
Cant be signed or unsigned and default can't be NULL if NULL is not allowed, which it is not